utils/key-finder/src/main.rs

```diff
@@ -0,0 +1,74 @@
+use std::env;
+use std::path::{MAIN_SEPARATOR, Path};
+use hex::encode;
+
+#[cfg(all(unix, not(target_os = "macos")))]
+const APPDATA: &'static str = r"/.local/share/";
+
+#[cfg(target_os = "macos")]
+const APPDATA: &'static str = r"/Library/Application Support/";
+
+#[cfg(target_os = "windows")]
+const APPDATA: &'static str = r"\AppData\Local\";
+
+#[cfg(not(any(unix, windows, target_os = "macos")))]
+compile_error!("Unsupported OS");
```

Comment:
You may just use [`sc_service::config::BasePath`](https://docs.subsocial.network/rust-docs/latest/sc_service/config/enum.BasePath.html) as we do it here for example:

https://github.com/gear-tech/gear/blob/master/node/cli/src/command.rs#L150

Reply:
Or even [`directories::ProjectDirs`](https://docs.rs/directories/latest/directories/struct.ProjectDirs.html) as `BasePath` does: https://docs.subsocial.network/rust-docs/latest/src/sc_service/config.rs.html#242-248